csharp-basics/exercises/Arithmetic/Exercise2-CheckOddEven/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Exercise2_CheckOddEven
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int numberToCheck = GetNumberToCheck();
+            string result = numberToCheck % 2 == 1 ? "Odd Number" : "Even Number";
+
+            Console.WriteLine($"---------\n {result}");
+            Console.WriteLine(" bye!");
+            Console.ReadKey();
+        }
+
+        static int GetNumberToCheck()
+        {
+            int valueToCheck;
+            Console.Write("Enter a number :: \n ");
+
+            while (!int.TryParse(Console.ReadLine(), out valueToCheck))
+            {
+                Console.Clear();
+                Console.WriteLine("You entered an invalid number!");
+                Console.Write("Enter a number :: \n ");
+            }
+            return valueToCheck;
```

Comment:
Kas notiek, ja nevar konvertēt? Paliek 0 vai ir null?

Reply:
Ja nevar konvertēt tad tad TryParse to valueToCheck neaiztiek & paliek ne-inicializēts~
(nezinu kāds ir tas hidden value ne-inicializētiem int 💭 , bet jebkurā gadījumā - tam while nevar tikt garām
kamēr neiedod derīgu inputu)

*googles* default value int ir 0

Reply:
Ja ir 0, tad ok. Ja būtu null, tad būtu errors